emuu-server/firebase/middleware.go

```diff
@@ -8,18 +8,9 @@ import (
 	"github.com/gin-gonic/gin"
 )
 
-// func corsAcceptance(w http.ResponseWriter, r *http.Request) {
```

Comment:
Good job for removing commented code. Must do a complete review once we merge to main to get rid of all dead code.